troposphere/templates/login.html

```diff
@@ -84,7 +83,7 @@
                 {% csrf_token %}
                 {% if disable_login == False %}
                 <a class="submitButton" href="{{ BASE_URL }}/login">
-                        Log in with your iPlant ID
+                        Log in with your {{ ORG_NAME }} ID
```

Comment:
Maybe this whole string should be a variable then. We can default to a value that concat's ORG_NAME this could be over written then if a special case calls for it or simply ignored if you have set ORG_NAME and the default is appropriate.